packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -835,6 +835,14 @@ export class DeltaManager
                 }
             }
 
+            if (to !== undefined && this.lastQueuedSequenceNumber >= to) {
+                // the client caught up while we were trying to fetch ops from storage
+                // bail out since we no longer need to request these ops
+                callback([]);
```

Comment:
I do not think making an empty call is required, or we do it in other places.

Reply:
The callback invokes `this.refreshDelayInfo(this.deltaStorageDelayId);`. If we skip this callback, that won't be called. Is that okay?

Reply:
I updated it to not pass anything so only `refreshDelayInfo` is invoked.

Reply:
It's important. I think it's cleaner for fetchMissingDeltas() to call it after it's done.

Reply:
I assume it still has to call `refreshDelayInfo` in every callback too?

Reply:
Yes